src/pages/utst%C3%A4llare.tsx

```diff
@@ -82,10 +90,37 @@ export default function Exhibitor() {
     setPendingChanges(false);
   }
 
-  return <>
-    <div className="mx-auto flex flex-col items-center text-center py-40">
-      <h1 className="md:text-5xl mx-10 text-3xl text-cerise mb-14">{exhibitor.data?.name}</h1>
-      {exhibitor.isLoading && <p className="text-cerise font-bold">Loading...</p>}
+  const [editState, setEditState] = useState(false);
+
+
+  return (
+    <>
+      <div className="mx-auto flex flex-col items-center py-40">
+        {/*Header*/}
+        <h1 className="uppercase text-cerise text-5xl font-medium text-center px-[10px] break-words">
+          {t.exhibitorSettings.header}
+        </h1>
+        {/*Header*/}
+
+        {/*Dropdown table*/}
+        <div className="h-full min-w-[200px] max-w-[1200px] w-full mt-[15px] px-[20px] min-[450px]:px-[60px] min-[704px]:px-[60px]">
+          {Table(
+            [
+              t.exhibitorSettings.table.row1.title,
+              t.exhibitorSettings.table.row2.title,
+              t.exhibitorSettings.table.row3.title,
+            ],
+            [],
+            [
+              <RowOne t={t} />,
+              <RowTwo t={t} exhibitorPackage="Baspaket" />,
+              <RowThree t={t} />,
+            ]
+          )}
+        </div>
+        {/*Dropdown table*/}
+
+        {/* {exhibitor.isLoading && <p className="text-cerise font-bold">Loading...</p>}
```

Comment:
We might as well remove this instead of commenting this out

Reply:
Though i guess that can be done when implementing this

Reply:
I've removed it for now.